docs/config.md

```diff
@@ -1163,92 +1157,6 @@ Default: `"5m"`
 
 Type: `duration`
 
-## `Pubsub`
```

Comment:
Please dont remove these! 

For things that remain, add `**DEPRECATED:**`  warning instead, like we did in the past for other things.
Then, when we remove, we remove content and leave "REMOVED" comment.
You can do REMOVED for parameters that are no longer wired up.

- DEPRECATED: https://github.com/ipfs/kubo/blob/master/docs/config.md#gatewaywritable
- REMOVED: https://github.com/ipfs/kubo/blob/master/docs/config.md#discoverymdnsinterval